internal/v1/handler_blueprints_test.go

```diff
@@ -122,6 +136,16 @@ func TestHandlers_UpdateBlueprint(t *testing.T) {
 	body["name"] = "Changing to correct body"
 	respStatusCodeNotFound, _ := tutils.PutResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/blueprints/%s", uuid.New()), body)
 	require.Equal(t, http.StatusNotFound, respStatusCodeNotFound)
+
+	// Test update customization users - invalid redacted password
+	body["customizations"] = map[string]interface{}{"users": []map[string]interface{}{{"name": "test", "password": "<REDACTED>"}}}
+	statusCode, _ = tutils.PutResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/blueprints/%s", result.Id), body)
```

Comment:
I thought this is ok, but this makes sure we do not override the password, but that actually means users can not update Blueprint without re-entering the password, right?

I think it's ok to have for now as MVP, but I assume that best case is:
When I update Blueprint with <redacted> as password, we pull the password from previous version and fail only if the password was not previously set.
I know it complicates the PR further, so I'm happy to just create a ticket for it and merge as is. Current solution is not best UX, but works and it's not a security concern :)

Reply:
I think separate issue would be nice, you can describe the use case there. I will be happy to implement it. Tbh I just though that whatever you give the update method it will just overwrites the blueprint, e.g. you had users and in update you just delete them. What if you want to change the users from password to ssh_key, that is allowed?

Reply:
I've tried my best here: https://issues.redhat.com/browse/HMS-4562